### PR TITLE
Added new AA config on CIFAR10 and ResNet18

### PR DESCRIPTION
**Pull request by nikita-savelyevv**
### Changes

- Added new Accuracy Aware config for quantization + pruning scenario on ResNet18 model and CIFAR10 dataset
- Added logic for downloading models

Results are given below. The AA metrics were averaged over 10 runs.
|   | Accuracy  | Compression rate  |  Duration |  
|---|---|---|---|
| Baseline FP32 | 88.61%  | 0%  | -  | 
| AA INT8 + Pruning|  87.78% | 58.13%  | 44.3 min  |

### Reason for changes

Existing AA configs run on ImageNet and take multiple days to finish which makes it hard for new users to get familiar with the approach and use it. The new config runs 45 mins on average with 8 workers on i9-10980XE CPU and single RTX 3090 GPU.

### CODE REVIEW

**Review by alexsu52 — APPROVED** [review]
Thanks for the contribution!

File: examples/torch/common/model_loader.py [hunk 1]
```diff
@@ -26,6 +30,9 @@
 from nncf.torch.utils import safe_thread_call
 
 
+MODELS_PATH = '~/.cache/nncf/models'
```
Comment (vshampor) [inline examples/torch/common/model_loader.py:33]: Bring this in line with the general approach in https://github.com/openvinotoolkit/nncf/pull/1579

**Review by vshampor — CHANGES_REQUESTED** [review]

File: examples/torch/common/model_loader.py (hunk 1 above)
Comment (nikita-savelyevv) [reply examples/torch/common/model_loader.py:33]: Seems like I need #1579 to be merged first. Maybe better to merge this PR and change `MODELS_PATH` in #1579 ?

**Review by vshampor — APPROVED** [review]
